cmake/Codegen.cmake

```diff
@@ -96,13 +106,20 @@ endfunction(GEN_XPU)
 
 GEN_XPU(
   native_functions.yaml
-  XPUFunctions.h
-  RegisterXPU.cpp
+ ${BUILD_TORCH_XPU_ATEN_GENERATED}/XPUFunctions.h
+ ${BUILD_TORCH_XPU_ATEN_GENERATED}/RegisterXPU.cpp
+ ${XPU_AOTI_INSTALL_DIR}/c_shim_xpu.h
+ ${XPU_AOTI_INSTALL_DIR}/c_shim_xpu.cpp
 )
 
 
-
+# The c_shim_xpu.cpp needs include files in ${CMAKE_BINARY_DIR}/xpu/ATen/ops/*.h)
+# The include path is auto generated as "#include <ATen/ops/*.h">
+# To follow the design of aoti codegen, here ${CMAKE_BINARY_DIR}/xpu is added to
+# $TORCH_XPU_OPS_INCLUDE_DIRS, so that "#include <ATen/ops/*.h>" works.
+list(APPEND TORCH_XPU_OPS_INCLUDE_DIRS ${CMAKE_BINARY_DIR}/xpu)
```

Comment:
Will the `<ATen/ops/*.h>` be ambiguous? Like we might have `native/ATen/ops/*.h`, or `cpu/ATen/ops.*.h`.

Reply:
The `c_shim_xpu.h/c_shim_xpu.cpp` and it's required headers `ATen/ops/*.h` is generated at the same script, so it always can find the dependency. we just match install dir of the torchgen.gen here.